onnxruntime/test/testdata/transform/runtime_optimization/qdq_convs_gen.py

```diff
@@ -0,0 +1,54 @@
+import onnx
+from onnx import helper
```

Comment:
This looks really familiar to me,  :smile: 
Can we use only one of this or the "onnxruntime/test/testdata/qdq_conv_test.py" and the generated onnx test model?

Reply:
updated to use a single generation script. keeping the model with three Convs to test out multiple runtime optimization records